vkcs/internal/clients/config.go

```diff
@@ -82,6 +82,9 @@ func ConfigureSdkProvider(d *schema.ResourceData, terraformVersion string) (Conf
 			IdentityEndpoint: getConfigParam(d, "auth_url", "OS_AUTH_URL", DefaultIdentityEndpoint),
 			UserDomainID:     getConfigParam(d, "user_domain_id", "OS_USER_DOMAIN_ID", ""),
 			UserDomainName:   getConfigParam(d, "user_domain_name", "OS_USER_DOMAIN_NAME", DefaultUserDomainName),
+			// Provide fake schema attribute name to do not add it to provider schema
+			// Provide empty default value to let gophercloud choose its default
+			EndpointType:     getConfigParam(d, "@interface", "OS_INTERFACE", ""),
```

Comment:
Wouldn't it be clearer just to take set it right from env as below?
```go
EndpointType: os.Getenv("OS_INTERFACE")
```

Reply:
done